metecho/api/models.py

```diff
@@ -379,25 +428,61 @@ class Meta:
         unique_together = (("repo_owner", "repo_name"),)
 
     def save(self, *args, **kwargs):
-        if not self.branch_name:
-            repo = gh.get_repo_info(
-                None, repo_owner=self.repo_owner, repo_name=self.repo_name
-            )
-            self.branch_name = repo.default_branch
-            self.latest_sha = repo.branch(repo.default_branch).latest_sha()
-
```

Comment:
@jerivas I couldn't figure out why we would ever want this second branch to run if the first one already ran?